package.json

```diff
@@ -121,17 +121,39 @@
     ]
   },
   "eslintConfig": {
+    "settings": {
+      "react": {
+        "version": "detect"
+      }
+    },
+    "extends": [
+      "plugin:react/recommended",
+      "plugin:react/jsx-runtime",
+      "plugin:testing-library/react"
+    ],
```

Comment:
Oh, I was surprised to see this here as these are React specific

Reply:
It is needed given that we have JSX in the tests.

Reply:
Yeah, but only on the /react package, right?

Reply:
Yea but liting shares a signle configuation.